plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/standard/StandardDeployJob.java

```diff
@@ -88,6 +89,8 @@ private CloudSdk getCloudSdk() {
     CloudSdk cloudSdk = new CloudSdk.Builder()
                           .addStdOutLineListener(new MessageConsoleWriterOutputLineListener(outputStream))
                           .addStdErrLineListener(new MessageConsoleWriterOutputLineListener(outputStream))
+                          .appCommandMetricsEnvironment(CloudToolsInfo.TOOLS_NAME_FOR_METRICS)
```

Comment:
I thought the same way. I guess it tries to say something like "set an environment variable for metrics and this is a command-line thing". There are other things like

`appCommandCredentialFile`
`appCommandOutputFormat`